### ISSUE TITLE
Java 10 - net.openhft.chronicle.core.Jvm: Unable to determine max direct memory

### ISSUE THREAD

**Issue by ospavel** [OpenHFT/Chronicle-Core#62]
In Java 10 code https://github.com/OpenHFT/Chronicle-Core/blob/c6b328e6e64ea371e556d41284e43f09d2aaab15/src/main/java/net/openhft/chronicle/core/Jvm.java#L459
throws exception (ignored in code):
```
java.lang.IllegalAccessException: class ##### cannot access class jdk.internal.misc.VM (in module java.base) because module java.base does not export jdk.internal.misc to unnamed module @###
	at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:360)
	at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:589)
	at java.base/java.lang.reflect.Method.invoke(Method.java:556)
```
Console error message: net.openhft.chronicle.core.Jvm: Unable to determine max direct memory

**Comment by epickrram** [comment]
Can you try running with the following JVM args:

```
--add-exports java.base/jdk.internal.ref=ALL-UNNAMED 
--add-exports java.base/jdk.internal.misc=ALL-UNNAMED 
--add-exports java.base/sun.nio.ch=ALL-UNNAMED
```

**Comment by ospavel** [comment]
Maybe issue can be solved by some changes in code? Otherwise this kind of parameters will be required to use in the all 9+ versions of Java...

**Comment by peter-lawrey** [comment]
There isn't any other way to get this information in Java 10.  The reflection method itself could be lazily loaded but you would still get a warning for one of the other methods which can't be lazily loaded.

**Comment by hft-team-city** [comment]
Released in Chronicle-Core-1.16.14 (https://github.com/OpenHFT/Chronicle-Core/releases/chronicle-core-1.16.14), BOM-1.16.125 (https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-1.16.125)

**Comment by peter-lawrey** [comment]
Hi,
  There was an explicit design decision by the JVM architects to require
these command line parameters for access to internal methods for which
there is no public API alternative in Java 11 afaik.

Regards,
   Peter.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Core-2.20.102](https://github.com/OpenHFT/Chronicle-Core/releases/chronicle-core-2.20.102), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
